### PR TITLE
m3utoxspf 0.9.1 (new formula)

### PR DESCRIPTION
**Pull request by ibrokemypie**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/m3utoxspf.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  def install
+    venv = virtualenv_create(libexec)
+    venv.pip_install_and_link "eyeD3"
+    libexec.install "m3utoxspf-#{version}.jar"
+    bin.write_jar_script libexec/"m3utoxspf-#{version}.jar", "m3utoxspf"
+  end
+
+  test do
+    system "curl", "http://www.jaymze.org/proglang/media/m3u/myaudio.m3u", "-o", "test.m3u"
```
Comment (bfontaine) [inline Formula/m3utoxspf.rb]: Please make this a `resource do` block instead of using `curl`.

**Review by bfontaine — CHANGES_REQUESTED** [review]

**Comment by bfontaine** [comment]
```
$ curl -sI http://selliott.org/sites/default/files/utilities/m3utoxspf/m3utoxspf-0.9.1.jar |grep Last
Last-Modified: Wed, 21 Nov 2012 19:56:42 GMT
```

Is this still maintained?

**Comment by ibrokemypie** [comment]
It was rather difficult to find, which may be why it is somewhat lacking in users, which in turn has resulted in a lack in development, however I think having it in homebrew at least until there is something that can replace it would be of value, save others from searching and maybe even help bring xspf to more people too

**Comment by bfontaine** [comment]
There’s also no {Debian,Ubuntu,Fedora,openSUSE,MacPorts,Fink} package for `m3utoxspf`. Quoting from [our rules for acceptable formulae](http://docs.brew.sh/Acceptable-Formulae.html#niche-or-self-submitted-stuff):

> The software in question must:
>
> * be maintained (e.g. upstream is still making new releases)
> * be known
> * be stable (e.g. not declared “unstable” or “beta” by upstream)
> * be used
> * have a homepage
>
> We will reject formulae that seem too obscure, partly because they won’t get maintained and partly because we have to draw the line somewhere.

I’m closing this for that reason. Feel free to maintain the formula [in your own tap](http://docs.brew.sh/How-to-Create-and-Maintain-a-Tap.html) however and re-submit it here once the software is popular. Thanks!

**Pull request closed without merging**